### ISSUE TITLE
Hi! style mixing issue!

### ISSUE THREAD

**Issue by scj0709** [eladrich/pixel2style2pixel#194]
Hello. 
I was impressed with your thesis and tried to train it. I trained with output_size=256.
And I want to do a style mixing test, but the output size is 256, how do I do style mixing?
It just runs and gives weird results.
![0](https://user-images.githubusercontent.com/40060713/133023141-a5135a3f-c872-4ccd-9233-ac3f72b947d8.png)

**Comment by yuval-alaluf** [comment]
Hi @scj0709 
What command did you run? Style mixing should work with a 256-resolution model. All that should be needed is to limit the latent mask to `latent_mask=8,9,10,11,12,13` and I believe it should work.

**Comment by scj0709** [comment]
![image](https://user-images.githubusercontent.com/40060713/133026705-6341d786-7252-458a-b1e9-786412230049.png)
Executed this command!

**Comment by yuval-alaluf** [comment]
I'll take a look. But it seems a bit weird to run style mixing for an inversion encoder since the task is not really suited for style mixing. Style mixing is better suited when there are many possible outputs for the input (e.g., input sketches)

**Comment by scj0709** [comment]
Using the pre-trained model at 1024x1024 you provided gives good results.
However, using a model trained separately at 256x256 does not produce the above result. Do I need to add additional code?
![0](https://user-images.githubusercontent.com/40060713/133051571-db8833e1-c21d-4c91-b9ce-b0fc3f580304.png)

**Comment by yuval-alaluf** [comment]
> However, using a model trained separately at 256x256 does not produce the above result. Do I need to add additional code?

It is strange that the style mixing is working fine with our pretrained 1024x1024 generator, but not with your generator, which looks to be returning reasonable reconstructions. 
I don't believe that additional code is needed here. 
The best thing I can recommend is trying to debug to see what happens. For example, you could run the `style_mixing` script and set the `--latent_mask=None`. Basically, this will turn off the style mixing and you should see the reconstruction image you posted above. 
This would help us investigate if there is a problem in the style mixing code or if there is something weird going on with your generator.
